src/platform/packages/shared/kbn-unified-metrics-grid/src/components/flyout/overview_tab.tsx

```diff
@@ -30,51 +30,7 @@ export const OverviewTab = ({ metric }: OverviewTabProps) => {
 
   return (
     <>
-      <TabTitleAndDescription metric={metric} />
-      {metric.source === 'otel' && metric.stability && (
```

Comment:
`stability` was never returned by the API. Therefore, this section was never rendered. Since we don't have any indication that this section will be part of the flyout, I decided to remove it